### ISSUE TITLE
 Category Design  Layout update="1 column" Display Mode="Static Block Only" Anchor="No" Does not fully remove sidebar.main

### ISSUE THREAD

**Issue by lano-vargas** [magento/magento2#12400]
In a category sidebar.main does not get removed It only removes sidebar.additional when Layout is set to 1 column.

1-install magento 2.1.7
2-create a category
3-update Design->Layout set to="1 column"
3.5- Display Settings->Display Mode->Static Block Only
3.5.1 Display Settings->Anchor-> set to="No"
4-create configurable product and create/add simple product variations e.g colour/size at least one.
5-navigate to magento category url should see nothing.
6- the still showing see screen shoot bellow
<img width="1261" alt="screen shot 2017-11-22 at 12 55 29" src="https://user-images.githubusercontent.com/3495481/33128526-9788c262-cf84-11e7-9c18-6b8cf46222e1.png">

6.5 The screen shoot bellow is what I see after adding 3.5 and 3.5.1

<img width="1255" alt="screen shot 2017-11-22 at 15 52 22" src="https://user-images.githubusercontent.com/3495481/33136503-48bad724-cf9d-11e7-8cba-ffd1f991e780.png">

7- I've tried adding Layout Update XML e.g 
   ```
<referenceContainer name="content">
     <referenceBlock name="sidebar.main" remove="true"/>
     <referenceBlock name="sidebar.additional" remove="true"/>
   </referenceContainer >

```
Still no luck!

Thanks

**Comment by magento-engcom-team** [comment]
@lano-vargas, thank you for your report.
This seems to be correct Magento behavior, 1 column design should not fully remove blocks, it should move them to 1 column. Please refer to the [Community Forums](https://community.magento.com/) or the [Magento Stack Exchange](https://magento.stackexchange.com/) site for advice or general discussion about this.
Otherwise, you may submit Pull Request with the suggested changes.

**Comment by lano-vargas** [comment]
@magento-engcom-team I don't thing this is acceptable as of per my xml update that's fine if is the default behaviour but 
why the xml update isn't removing the side.main ? By the way I've already used the community and it seems that the xml is ok 
ref: https://magento.stackexchange.com/questions/202712/magento-2-remove-sidebar-main-sidebar-aditional-with-xml-update-does-not-work#202712

**Comment by lano-vargas** [comment]
@magento-engcom-team How about set to  "static block" it should remove the the sidebar.main block.
This is most definitely a bug.
I can replicate the same issue an this a related bug
ref: https://github.com/magento/magento2/issues/7765

**Comment by lano-vargas** [comment]
@magento-engcom-team If is correct Magento behavior when set to one "1 column" not to fully remove blocks then I can only suggest that when Anchor is set to "No"  something in: 
`magento2/app/code/Magento/Catalog/view/frontend/templates/navigation/left.phtml`
include:

> if ($_count && $block->getCategory()->getDisplayMode()  !== \Magento\Catalog\Model\Category::DM_PAGE): ....

**Comment by magento-engcom-team** [comment]
@lano-vargas Thank you for clarifying some steps. We tried to follow them and we got an empty category page without products and without the filter navigation. Could you please watch the [video](https://streamable.com/wh0v9) we made where we are trying to follow the steps you provided. And tell us if we missing something. Thank you in advance. https://streamable.com/wh0v9

**Comment by magento-engcom-team** [comment]
@lano-vargas, we are closing this issue due to inactivity. If you'd like to update it, please reopen the issue.

**Comment by lano-vargas** [comment]
@magento-engcom-team  Thanks for the video.
I can tell you that you are missing the version in question have this bug been fix or not exist in 2.2.2 ?
The version you should be testing is 2.1.7
Thanks.
